src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -510,18 +516,24 @@ public void Visit(SensitivityAnalysis sensitivityAnalysis)
 
       public void Visit(ParameterIdentification parameterIdentification)
       {
+         _currentlyActivePI = parameterIdentification.Id;
+         _parameterIdentificationRunning = _activePIs.Contains(parameterIdentification.Id);
          updateParameterIdentificationItems(parameterIdentification);
+         _eventPublisher.PublishEvent(new ParameterIdentificationSelectedEvent(parameterIdentification));
       }
 
       public void Handle(ParameterIdentificationStartedEvent parameterIdentificationEvent)
       {
+         _activePIs.Add(parameterIdentificationEvent.ParameterIdentification.Id);
          _parameterIdentificationRunning = true;
          updateParameterIdentificationItems(parameterIdentificationEvent.ParameterIdentification);
       }
 
       public void Handle(ParameterIdentificationTerminatedEvent parameterIdentificationEvent)
       {
-         _parameterIdentificationRunning = false;
+         _activePIs.Remove(parameterIdentificationEvent.ParameterIdentification.Id);
+         if (parameterIdentificationEvent.ParameterIdentification.Id == _currentlyActivePI)
+            _parameterIdentificationRunning = false;
```

Comment:
this is wrong here. One might still be running you have multiple ones. 
What are you trying to do ? Is this to only have a stop button? This is also weird that the menu and tool bar presenter needs to track active PIs were we already have a manager tracking active PIs...

Reply:
Now we have multiple PIs. So we have the list and the active one. When the event `ParameterIdentificationTerminatedEvent` is handled, then its id is removed from the list and in case it was the current PI, the running flag is set to false (so we can show the run button enabled, and disabled the stop for example). Sorry if it is not clear. I am open to suggestions on how to improve it.